addons/sourcemod/scripting/MyJailbreak/duckhunt.sp

```diff
@@ -59,6 +61,8 @@ char huntermodel[256] = "models/player/custom_player/legacy/tm_phoenix_heavy.mdl
 char g_sCustomCommand[64];
 char g_sEventsLogFile[PLATFORM_MAX_PATH];
 char g_sAdminFlag[32];
+char g_sModelPathCT[256];
```

Comment:
char g_sModelPathCT[MAXPLAYERS+1][256];
char g_sModelPathT[MAXPLAYERS+1][256];

to store different playermodels to each client

GetEntPropString(client, Prop_Data, "m_ModelName", g_sModelPathCT[client], sizeof(g_sModelPathCT[client]));
GetEntPropString(client, Prop_Data, "m_ModelName", g_sModelPathT[client], sizeof(g_sModelPathT[client]));

SetEntityModel(client, g_sModelPathCT[client]);
SetEntityModel(client, g_sModelPathT[client]);

so every player get **his** old model and not the model of the last looped client